spark-extension-shims-spark3/src/main/scala/org/apache/spark/sql/execution/blaze/shuffle/uniffle/BlazeUniffleShuffleReader.scala

```diff
@@ -107,6 +116,7 @@ class BlazeUniffleShuffleReader[K, C](
       val emptyPartitionIds = new util.ArrayList[Int]
       for (partition <- startPartition until endPartition) {
         if (partitionToExpectBlocks.get(partition).isEmpty) {
+          logInfo(s"$partition is empty partition")
```

Comment:
`logInfo(s"$partition is empty")
`
or 

`logInfo(s"$partition is a empty partition")`